src/app/csc/csc.component.ts

```diff
@@ -283,12 +283,13 @@ export class CscComponent implements OnInit {
         if (
           this.cscProjectsList[project].contacts.principal_investigators != null
         ) {
-          this.cscProjectsList[project].investigators_formatted = "";
+          this.cscProjectsList[project].investigators_formatted = "<ul>";
           for (const pi of this.cscProjectsList[project].contacts
             .principal_investigators) {
             this.cscProjectsList[project].investigators_formatted +=
-              pi.name + "&nbsp;<i>(" + pi.organization + "</i>)<br>";
+              "<li>" + pi.name + "<i>(" + pi.organization + "</i>)</li>";
```

Comment:
Missing the space here between PI name / institution

Reply:
Added back in the space, thanks for catching that!